WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Models/DashboardCard.swift

```diff
@@ -147,15 +147,21 @@ enum DashboardCard: String, CaseIterable {
         }
     }
 
-    func shouldShow(
+    static func shouldShowDynamicCard(
         for blog: Blog,
-        dynamicCardPayload: DashboardDynamicCardModel.Payload,
+        payload: DashboardDynamicCardModel.Payload,
+        remoteFeatureFlagStore: RemoteFeatureFlagStore,
         isJetpack: Bool = AppConfiguration.isJetpack
     ) -> Bool {
-        return self == .dynamic
-        && isJetpack
+        let remoteFeatureFlagEnabled = {
+            guard let key = payload.remoteFeatureFlag else {
+                return true
+            }
+            return remoteFeatureFlagStore.value(for: key) ?? false
```

Comment:
I think if the key is not found in the remoteFeatureFlag cache, meaning we didn't add an entry to the feature flags endpoint, we should default to showing the card, not hiding it.
```suggestion
            return remoteFeatureFlagStore.value(for: key) ?? true
```

Reply:
Following an internal, we've agreed to default to `false`